pyscriptjs/tests/integration/test_zz_examples.py

```diff
@@ -105,32 +109,67 @@ def test_bokeh_interactive(self):
         assert self.page.title() == "Bokeh Example"
         wait_for_render(self.page, "*", '<div.*?class=\\"bk\\".*?>')
 
-    @pytest.mark.xfail(reason="Flaky test #759")
     def test_d3(self):
-        # XXX improve this test
         self.goto("examples/d3.html")
         self.wait_for_pyscript()
         assert (
             self.page.title() == "d3: JavaScript & PyScript visualizations side-by-side"
         )
         wait_for_render(self.page, "*", "<svg.*?>")
+        assert "PyScript version" in self.page.content()
+        pyscript_chart = self.page.wait_for_selector("#py")
+
+        # Let's simply assert that the text of the chart is as expected which
+        # means that the chart rendered successfully and with the right text
+        assert "🍊21\n🍇13\n🍏8\n🍌5\n🍐3\n🍋2\n🍎1\n🍉1" in pyscript_chart.inner_text()
 
     def test_folium(self):
-        # XXX improve this test
         self.goto("examples/folium.html")
         self.wait_for_pyscript()
         assert self.page.title() == "Folium"
         wait_for_render(self.page, "*", "<iframe srcdoc=")
+        # assert "Unemployment Rate (%)" in self.page.content()
```

Comment:
we can remove this comment? since we check it below?

Reply:
Oops thanks! Completly missed this one!